packages/flutter_tools/lib/src/web/chrome.dart

```diff
@@ -222,6 +222,8 @@ class ChromiumLauncher {
     if (process != null && cacheDir != null) {
       unawaited(process.exitCode.whenComplete(() {
         _cacheUserSessionInformation(userDataDir, cacheDir);
+        // cleanup temp dir
+        userDataDir.deleteSync(recursive: true);
```

Comment:
Can you catch FileSystemExceptions here, as its not really actionable?

```suggestion
        try {
          userDataDir.deleteSync(recursive: true);
        } on FileSystemException {
          // ignore
        }
```